app/lib/active_record_utils.rb

```diff
@@ -12,8 +12,9 @@ def self.with_transaction_and_rescue(audit_results)
     rescue ActiveRecord::RecordNotFound => e
       audit_results.add_result(AuditResults::DB_OBJ_DOES_NOT_EXIST, e.inspect)
     rescue ActiveRecord::ActiveRecordError => e
+      Rails.logger.error("db update error; more context to follow: #{e.inspect}: #{e.backtrace.inspect}")
```

Comment:
we want to log the backtrace in the Rails logs;  this will do that, and then the additional contextual information (which check being run?  which storage root?)  will come from AuditResults.report_results().